scripts/pancancer.py

```diff
@@ -0,0 +1,341 @@
+                                'name':'seqware_whitestar_pancancer',
+                                'image_name': 'pancancer/seqware_whitestar_pancancer:1.1.1'
+                            }
+                        },
+                        'ami_id':'ami-12345',
+                        'default-ini':'http://something.ini',
+                        'instance-type':'m3.medium',
+                        'lvm_devices':'/dev/xvdb,/dev/xvdc,/dev/xvdd,/dev/xvde'
+                    },
+                    'DKFZ_EMBL':
+                    {
+                        'full_name':'Workflow_Bundle_DEWrapperWorkflow_1.0.5_SeqWare_1.1.1',
+                        'http_workflow':
+                        {
+                            'url':'https://s3.amazonaws.com/oicr.workflow.bundles/released-bundles/Workflow_Bundle_DEWrapperWorkflow_1.0.5_SeqWare_1.1.1.zip',
+
+                            'version':'1.0.5'
+                        },
+                        'containers':
+                        {
+                            'pcawg-delly-workflow': {
+                                'name':'pcawg-delly-workflow',
+                                'image_name': 'pancancer/pcawg-delly-workflow:1.0'
+                            },
+                            'seqware_whitestar_pancancer': {
+                                'name':'seqware_whitestar_pancancer',
+                                'image_name': 'pancancer/seqware_whitestar_pancancer:1.1.1'
+                            }
+                        },
+                        's3_containers':
+                        {
+                            'dkfz_dockered_workflows':
+                            {
+                                'name':'dkfz_dockered_workflows',
+                                'url':'https://s3.amazonaws.com/oicr.docker.private.images/dkfz_dockered_workflows_1.3.tar'
+                            }
+                        },
+                        'ami_id':'ami-12345',
+                        'default-ini':'http://something.ini',
+                        'instance-type':'m3.medium',
+                        'lvm_devices':'/dev/xvdb,/dev/xvdc,/dev/xvdd,/dev/xvde'
+                    }
+                }
+
+    @staticmethod
+    def get_workflow_names():
+        keys = ''
+        for k in WorkflowLister._workflows:
+            keys += k+'\n'
+        return keys
+
+    @staticmethod
+    def get_workflow_details(workflow_name):
+        return WorkflowLister._workflows[workflow_name]
+
+###
+
+class Workflows(Command):
+    "This command  can help you configure and select workflows."
+    log = logging.getLogger(__name__)
+
+    def get_parser(self,prog_name):
+        parser = super(Workflows,self).get_parser(prog_name)
+        #parser.add_mutually_exclusive_group()
+        workflows_subparser = parser.add_subparsers(title='subcommands',help='workflows subcommands: list and config',dest='subparser_name')
```

Comment:
Style wise, you should limit your lines to 80 characters to comply with the pep standard, truncate at 80 lines and use brackets to complete strings or function calls:
	ie func(sdfadsfsd,
		    sdfsfdsdfs
		   )
        or ("sfdfsdfsd"
             "sdfsdfs"
             )